hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/NodesConfig.java

```diff
@@ -14,4 +31,13 @@ public record NodesConfig(
         @ConfigProperty(defaultValue = "true") @NetworkProperty boolean gossipFqdnRestricted,
         @ConfigProperty(defaultValue = "true") @NetworkProperty boolean enableDAB,
         @ConfigProperty(defaultValue = "253") @NetworkProperty int maxFqdnSize,
-        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean updateAccountIdAllowed) {}
+        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean updateAccountIdAllowed,
+        /* Node rewards HIP-1064 configurations */
+        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean nodeRewardsEnabled,
+        @ConfigProperty(defaultValue = "0") @NetworkProperty long minNodeRewardUsd,
```

Comment:
Perhaps `minPerPeriodNodeRewardUsd` will be a cleaner distinction with the `targetUsdNodeRewards`?

Reply:
Yes, changed as suggested. Also changed `targetNodeRewardsUsd` to `targetYearlyNodeRewardsUsd`